src/core/compact_object.cc

```diff
@@ -1008,7 +1008,8 @@ void CompactObj::Free() {
     u_.json_obj.json_ptr->~JsonType();
     tl.local_mr->deallocate(u_.json_obj.json_ptr, sizeof(JsonType), kAlignSize);
   } else if (taglen_ == SBF_TAG) {
-    DeleteMR<SBF>(u_.sbf);
+    u_.sbf->~SBF();
```

Comment:
Thanks, but why did you decide to fix it like this instead of fixing the code in DeleteMR? 
which is clearly a bug, btw.

Reply:
I did not notice that DeleteMR is a wrapper so I did not look the implementation. Fixed.